dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -148,42 +106,72 @@ RUN readlink -f {{ $beatBinary }} | xargs setcap {{ .linux_capabilities }}
 {{- end }}
 
 {{- if ne .user "root" }}
-RUN groupadd --gid 1000 {{ .BeatName }}
-RUN useradd -M --uid 1000 --gid 1000 --groups 0 --home {{ $beatHome }} {{ .user }}
-{{- if (and (eq .BeatName "heartbeat") (not (contains .from "ubi-minimal")))  }}
-RUN chown {{ .user }} $NODE_PATH
-{{- end }}
+    RUN groupadd --gid 1000 {{ .BeatName }}
+    RUN useradd -M --uid 1000 --gid 1000 --groups 0 --home {{ $beatHome }} {{ .user }}
+
+    {{- if (and (eq .BeatName "heartbeat") (not (contains .from "ubi-minimal"))) }}
+        ENV NODE_PATH={{ $beatHome }}/.node
+        RUN echo \
+            $NODE_PATH \
+            {{ $beatHome }}/.config \
+            {{ $beatHome }}/.synthetics \
+            {{ $beatHome }}/.npm \
+            {{ $beatHome }}/.cache \
+            | xargs -IDIR sh -c 'mkdir -p DIR && chmod 0770 DIR'
+
+        # Setup synthetics env vars
+        ENV ELASTIC_SYNTHETICS_CAPABLE=true
+        ENV SUITES_DIR={{ $beatHome }}/suites
+        ENV NODE_VERSION=16.15.0
+        ENV PATH="$NODE_PATH/node/bin:$PATH"
+        # Install the latest version of @elastic/synthetics forcefully ignoring the previously
+        # cached node_modules, heartbeat then calls the global executable to run test suites
+        # Setup node
+        # Also try to add any deps that were missed earlier in terms of playwright reqs
+        # playwright install-deps is kept up to date with additional packages chromium needs
+        # beyond the base packages listed above
+        RUN cd /usr/share/heartbeat/.node \
+        && NODE_DOWNLOAD_URL="" \
+        && case "$(arch)" in \
```

Comment:
There is a set of Environment variables defined by Docker to detect the architecture, also this code is not need if you use the Debian package. 

```
BUILDPLATFORM — matches the current machine. (e.g. linux/amd64)
BUILDOS — os component of BUILDPLATFORM, e.g. linux
BUILDARCH — e.g. amd64, arm64, riscv64
BUILDVARIANT — used to set ARM variant, e.g. v7
TARGETPLATFORM — The value set with --platform flag on build
TARGETOS - OS component from --platform, e.g. linux
TARGETARCH - Architecture from --platform, e.g. arm64
```

Reply:
Awesome, given that this code mas merely reformatted, not changed. Could we save that for a different PR? We don't use the debian packages because they don't add any value here and add complexity since we'd need to setup the PPA etc. since the ubuntu version may trail mainstream node releases.